### PR TITLE
feat(apps): Handle same and older app correctly, show installed apps

### PR DESCRIPTION
**Pull request by tfenster**
- Better handling for cases where we install artifacts that have previously been installed with the same or older version
- Show installed apps on startup, especially interesting when using .baks

### CODE REVIEW

File: artifacts/AdditionalSetupArtifacts.ps1 [hunk 1]
```diff
@@ -166,6 +166,10 @@ $properties = @{}
 
 Invoke-LogEvent -name "AdditionalSetup - Started" -telemetryClient $telemetryClient
 
+# Show installed apps
+Write-Host "##[group]Initially installed apps"
+Get-NAVAppInfo -Tenant default -TenantSpecificProperties -ServerInstance BC
```
Comment (lippertmarkus) [inline artifacts/AdditionalSetupArtifacts.ps1]:
```suggestion
Get-NAVAppInfo -Tenant $tenantId -TenantSpecificProperties -ServerInstance $ServerInstance
```

**Review by lippertmarkus — CHANGES_REQUESTED** [review]

**Review by lippertmarkus — CHANGES_REQUESTED** [review]
some nits

**Review by lippertmarkus — CHANGES_REQUESTED** [review]
some nits

File: artifacts/AdditionalSetupArtifacts.ps1 (hunk 1 above)
Comment (tfenster) [reply artifacts/AdditionalSetupArtifacts.ps1]: @lippertmarkus Thanks! Should now be fixed. Other than that, all good?

**Review by lippertmarkus — APPROVED** [review]

**Pull request merged**